jboss-kie-common/added/launch/jboss-kie-common.sh

```diff
@@ -6,16 +6,20 @@
 query_ocp_api() {
     local ocpApi="${1}"
     local resourcePath="${2}"
+    local namespaceEnv=$(cat ${NAMESPACE}/namespace)
+    local namespace="${namespaceEnv:-/var/run/secrets/kubernetes.io/serviceaccount/namespace}"
+    local tokenEnv=$(cat ${TOKEN}/token)
+    local token="${tokenEnv:-/var/run/secrets/kubernetes.io/serviceaccount/token}"
+    local caCertEnv=${CACERT}/ca.crt
+    local cacert="${caCertEnv:-/var/run/secrets/kubernetes.io/serviceaccount/ca.crt}"
     # only execute the following lines if this container is running on OpenShift
-    if [ -e /var/run/secrets/kubernetes.io/serviceaccount/token ]; then
```

Comment:
why remove it?
it is intended to run only on ocp/k8s.
I don't see a reason for updating this function.

Reply:
How can we write a mock test if the function can't be tested outside ocp ?

Reply:
using the mock server to mock these requests?
we do this on the hooks tests, take a look there.